lib/runtime-core/src/vm.rs

```diff
@@ -401,6 +401,21 @@ impl Ctx {
         }
     }
 
+    /// Splits the borrow, getting both Memory and a mutable reference to
```

Comment:
It's required by the Rust borrow checker; it's a split borrow in that one borrow comes in and 2 come out that otherwise is not possible with safe Rust.  There's a function for splitting borrows over slices I believe `&mut [u8] -> (&mut [u8], &mut [u8])` which is where I got the terminology from